Consul/Properties/AssemblyInfo.cs

```diff
@@ -1,18 +1,5 @@
-using System.Reflection;
 using System.Runtime.CompilerServices;
 
-// General Information about an assembly is controlled through the following
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
-[assembly: AssemblyTitle("Consul.NET")]
-[assembly: AssemblyDescription(".NET API for Consul (http://www.consul.io/)")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("G-Research Limited")]
-[assembly: AssemblyProduct("Consul.NET")]
-[assembly: AssemblyCopyright("Copyright 2015 PlayFab, Inc.; Copyright 2020 G-Research Limited")]
```

Comment:
I think that we should add the `Copyright` element to the props file to avoid loosing this.

Reply:
Alrighty, done